pom.xml

```diff
@@ -56,6 +58,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>com.squareup.okio</groupId>
+      <artifactId>okio</artifactId>
+      <version>2.4.0</version>
```

Comment:
Why is this here? Is it an unexpressed dependency of some existing direct dependency?